_posts/2017-09-05-planning-for-7.0.md

```diff
@@ -0,0 +1,498 @@
+> Change our publish process (probably through Lerna) to publish the packages in `experimental/` independently.
+
+## 💀 Possible Deprecations
+
+### Deprecate the "env" option in `.babelrc`
+
+> [babel/babel#5276](https://github.com/babel/babel/issues/5276)
+
+The "env" configuration option (not to be confused with babel-preset-env) has been a source of confusion for our users as seen by the numerous issues reported.
+
+The [current behavior](http://babeljs.io/docs/usage/babelrc/#env-option) is to merge the config values into the top level values which isn't always intuitive such that developers end up putting nothing in the top level and just duplicating all the presets/plugins under separate envs.
+
+To eliminate the confusion (and help our power users), we're considering dropping the env config option all together and recommending users use the proposed JS config format (see below).
+
+### Deprecate ES20xx presets
+
+> We already deprecated preset-latest a while ago, and ES2016/ES2017 in https://twitter.com/left_pad/status/897483806499885057
+> It's annoying making a yearly preset (extra package/dependency, issues with npm package squatting unless we do scoped packages)
+
+Developers shouldn't even need to make the decision of what yearly preset to use? If we drop/deprecate these presets then everyone can use [babel-preset-env](https://github.com/babel/babel-preset-env) instead which will already update as the spec changes.
+
+## 🤔 Questions
+
+### ~~Deprecate Stage X presets~~
+
+> Ok we'll probably keep these and just make major version bumps
+> [babel/babel#4914](https://github.com/babel/babel/issues/4914)
+> [babel/babel#5128](https://github.com/babel/babel/issues/5128)
+
+Many in the community (and TC39) have expressed concerns over the Stage X presets. I believe I just added them to have an easy migration path from Babel 5 to Babel 6 (used to be a "stage" option).
+
+While we want to have an easy to use tool, it turns out that many companies/developers use these "not yet JavaScript" presets all the time, and in production. "Stage 0" doesn't really set the same tone as `babel-preset-dont-use-this-stage-0`.
+
+> Ariya just made an [awesome poll](https://twitter.com/AriyaHidayat/status/833797322786025472) that explains what I'm talking about
+
+Developers don't actually know what features are in what version of JavaScript (and they shouldn't have to know). However it is a problem when we all start thinking that "features" that are actually still proposals are in the spec already.
+
+Many open source projects (including Babel still 😝), tutorials, conference talks, etc all use `stage-0`. React promotes the use of JSX, class properties (currently Stage 2), object rest/spread (now Stage 3) and we all believe that it's just JavaScript because Babel compiled it for them. So maybe removing this abstraction would help people understand more about what is going on and the tradeoffs one is making when choosing to use Stage X plugins.
```

Comment:
`"class properties (currently Stage 2)"` should update this to say stage 3?

Reply:
Yeah that part was copied over from the first post in feb, and we aren't deprecating so I forgot to check.